### PR TITLE
python: add Multi/Empty unlocks

### LINKED ISSUE #1591: Python: add Multi/Empty Unlock
Counterpart of https://github.com/iotaledger/iota-sdk/pull/1590

### PR DESCRIPTION
**Pull request by thibault-martinez**
Closes https://github.com/iotaledger/iota-sdk/issues/1591

### CODE REVIEW

**Review by Thoralf-M — DISMISSED** [review]

File: bindings/python/iota_sdk/types/unlock.py [hunk 1]
```diff
@@ -84,11 +88,36 @@ class NftUnlock:
     type: int = field(default_factory=lambda: int(UnlockType.Nft), init=False)
 
 
+@json
+@dataclass
+class MultiUnlock:
+    """Unlocks a MultiAddress with a list of other unlocks.
+    """
+    unlocks: List[Unlock]
```
Comment (kwek20) [inline bindings/python/iota_sdk/types/unlock.py]: doesnt this need 
```
= field(
        metadata=config(
            decoder=deserialize_unlock
        ))
```

?

Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/unlock.py]: Hmm idk, we don't have it in the unlocks of a payload

Comment (kwek20) [reply bindings/python/iota_sdk/types/unlock.py]: we have it on most other inheriting types, i think we need it.
Can you add it to the other one as well then?

Comment (thibault-martinez) [reply bindings/python/iota_sdk/types/unlock.py]: Done

**Review by kwek20 — APPROVED** [review]

**Pull request merged**